Monika After Story/game/script-topics.rpy

```diff
@@ -4820,3 +4820,196 @@ label monika_asks_family:
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_prom",category=['school'],prompt="Prom",random=True))
+
+label monika_prom:
+    m 2c "You know [player]?"
+    m 1c "Sometimes I wish the game would have lasted longer."
+    m 1d "This game was made for to end once one of the girls confesses to you after the festival."
+    m "Then what do you know. Game over."
+    m 1o "And the whole game lets you start from scratch again as if nothing ever happened."
+    m 3c "Most visual novels only last a few months then go back to the beginning as soon as the player wants to do a new route with another girl."
+    m 3m "But thanks to my tampering, the game '{i}ended{/i}' before the festival even started."
+    m 3n "Well, it {i}did{/i} bring us together so I can't complain."
+    m 2b "But still..."
+    m 2f "Sometimes I wish it would have lasted the whole year. Not just the month leading up to the festival."
+    m 3a "We could've enjoyed other events like sports festivals, christmas parties, field trips."
+    m 3e "And maybe even prom..."
+    m 2e "I would've loved to have been your prom date."
+    m 1e "What I wouldn't give to get the opportunity..."
+    m 1q "..."
+    m 1l "But I guess the game would never let me go that far."
+    m 1a "Speaking of prom. [player], have you ever gone to prom?"
+    menu:
+        "I have.":
+            m 1a "Oh? How was it?"
+            menu:
+                "It was pretty fun.":
+                    m 1b "That's great!"
+                    m 1j "I've always heard that prom's are so much fun!"
+                    m "Everyone in the school getting together and enjoying themselves."
+                    m 2j "It just sound like a blast!"
+                    m 2m "Though I am kinda jealous of you [player]."
+                    m 2l "{i}ahaha.{/i}"
+                    m 3b "But believe it or not [player] you might not believe prom started like that."
+                "It was bad.":
+                    m 1f "Is that so, [player]?"
+                    m 2e "I guess prom's not for everybody so you shouldn't blame yourself for not having too much fun."
+                    m 3a "Maybe if I was there, you would have enjoyed it more."
+                    m 3k "{i}ahaha~{/i}"
+                    m 3b "Here's a fun little tidbit about prom. You might find it ineresting."
+                "It would've been better if you were there.":
+                    m 2e "Awww, that so sweet [player]."
+                    m 3b "I'm sure if we were together, it would have been ten times better!"
+                    m 3k "{i}ahaha~{/i}"
+                    m 2b "By the way."
+                    m 3b "Did you know how prom started [player]?"
+        "Not really.":
+            m 2d "Oh. Does that mean there's one coming up?"
+            menu:
+                "Yes.":
+                    m 1c "Are you going [player]?"
+                        menu:
```

Comment:
This menu indentation is wrong, it should be at the same level of `m 1c "Are you going [player]?"`, with this change you'll also have to fix the indentation of all this menu branch